onnxruntime/test/onnx/main.cc

```diff
@@ -299,8 +311,13 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       {"BatchNorm2d_momentum_eval", "disable reason"},
       {"BatchNorm3d_eval", "disable reason"},
       {"BatchNorm3d_momentum_eval", "disable reason"},
+#if defined(__GNUG__) && !defined(__LP64__)
       {"constantofshape_float_ones", "test data bug"},
       {"constantofshape_int_zeros", "test data bug"},
+#else
+      {"constantofshape_float_ones", "test data bug", {"9e55ace55aad1ada27516038dfbdc66a8a0763db","7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef"}},
```

Comment:
>{"9e55ace55aad1ada27516038dfbdc66a8a0763db","7d7bc83d29a328233d3e8affa4c4ea8b3e3599ef" [](start = 54, length = 86)

This not good. Talk to me before you proceed any further.

Reply:
Switched to tagging.